.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+    "extends":["airbnb"],
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true
+    },
+    "rules": {
+        "linebreak-style":"off",
+        "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+        "react/button-has-type": [1, {
+            "button": true,
+            "submit": true,
+            "reset": true
+          }],
+          "react/prop-types": "off",
```

Comment:
At some point we will get rid of this and incorporate prop-types and destructuring assignment in the project.
For now it's ok.

Reply:
Okey.I'm going to read more about eslint.